### ISSUE TITLE
Zoom functionality for occlusion editor

### ISSUE THREAD

**Issue by hgiesel** [hgiesel/closet#38]
* Add a slider option to make the picture bigger during occlusion creation:
  * Set zoom style attribute on container
* Prevent labels from grow when using `zoom`: Otherwise they get so big, you can't see anything.

**Comment by kleinerpirat** [comment]
You could also put an image editor into a context menu through a QWidgetAction like so:

```
menu = QMenu()

image_editor = SomeQWidget()
widget_action = QWidgetAction()

widget_action.setDefaultWidget(image_editor)
menu.addAction(widget_action)

```
One advantage would be that you can always edit the image occlusion, no matter the scroll position. The other advantage would be that you could restrict the maximum image height in the fields (better overview) without compromising the image occlusion functionality.

Here is an example of this from an unpublished addon of mine:

![1dcd43ad-9ce1-460e-b607-be79bb5b4a3b](https://user-images.githubusercontent.com/62722460/100613607-1be11e00-3315-11eb-9274-b72b6be7df83.gif)

**Comment by hgiesel** [comment]
It does look polished. However one common feedback I got for the IO functionality was that they liked it because it did not create a new window (unlike Glutanimate's IO Enhanced), which makes it feel less intrusive. While this would not be a new window, it would be another widget container. I think some of my users would dislike switching to a widget like that.

One reason I created this issue, because I had [this add-on](https://ankiweb.net/shared/info/229181581) set with way too small dimensions, making pictures way too small. Maybe I will recreate the functionality of that add-on within Closet, and make it configurable via the context menu.

**Comment by kleinerpirat** [comment]
I understand your point about having too many windows and I count myself as one of those users who like having everything in the add window without any extra windows popping up.

But I use that max-image height addon too and living without it is suboptimal, which is why I am interested in this issue ;)

In my case I went for that context menu method because it was the least intrusive of all the options i had (still an extra window though).

Definitely excited about that zoom slider and curious how it will turn out!

**Comment by hgiesel** [comment]
Zoom Slider was implemented in v0.3.0
